source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -661,8 +884,75 @@ The XML excerpt below is an example of a compatible XML data feed.
 </mywebstore>
 ~~~
 
+### XML with Size Variations
```

Comment:
[suggestion] I would like it if "Single XML" and "XML with Size Variations" were different "tabs" so that you can view only one at a time.. Same goes for the XSD.